catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/impl/FileSystemPersistenceProvider.java

```diff
@@ -173,23 +137,12 @@ Object loadFromPersistence(String key) {
     return values;
   }
 
-  private FilenameFilter getFilenameFilter() {
-    FilenameFilter filter =
-        new FilenameFilter() {
-          @Override
-          public boolean accept(File file, String name) {
-            return name.toLowerCase(Locale.getDefault()).endsWith(SER);
-          }
-        };
-    return filter;
-  }
-
   @Override
   public Set<String> loadAllKeys() {
     Set<String> keys = new HashSet<String>();
     LOGGER.debug("Entering loadAllKeys");
 
-    File[] files = new File(getMapStorePath()).listFiles(getFilenameFilter());
+    File[] files = persistencePath.toFile().listFiles(getFilenameFilter());
```

Comment:
❓Should we consider using `Files.list(persistentPath).filter(this::serializedFiles)` instead?

Reply:
Changed